src/components/ListOfTipsAndComments.vue

```diff
@@ -167,13 +167,14 @@ export default {
     },
     reloadData() {
       this.showLoading = true;
-      Backend.getAllComments()
-        .then((allComments) => {
+      Backend.getUserComments(this.address)
+        .then((userComments) => {
           this.showLoading = false;
           this.error = false;
-          this.comments = allComments.filter(
-            (comment) => comment.author === this.address,
-          );
+          if (userComments.length) {
+            this.comments = userComments
+              .sort((prev, next) => new Date(next.createdAt) - new Date(prev.createdAt));
```

Comment:
`sort` accepts comparator, it should compare two values, but there is no guarantee that the first one will be before the second one. So, I would name them just `a`, `b` 😅